src/member/apps.py

```diff
@@ -3,3 +3,9 @@
 
 class MemberConfig(AppConfig):
     name = 'member'
+
+    def ready(self):
+        # import signal handlers
+        # (every models imported inside handlers will be instantiated as
+        # soon as the registry is fully populated.)
+        import member.signals.handlers  # noqa
```

Comment:
Let me understand @lingxiaoyang,  your point is change `@receiver(post_delete, sender=ClientScheduledStatus)` by `post_delete.connect(sender=ClientScheduledStatus)` ??